keras_cv/models/efficientnet_v1.py

```diff
@@ -0,0 +1,733 @@
+            )(se)
+            x = layers.multiply([x, se], name=name + "se_excite")
+
+        # Output phase
+        x = layers.Conv2D(
+            filters_out,
+            1,
+            padding="same",
+            use_bias=False,
+            kernel_initializer=CONV_KERNEL_INITIALIZER,
+            name=name + "project_conv",
+        )(x)
+        x = layers.BatchNormalization(axis=BN_AXIS, name=name + "project_bn")(x)
+        if id_skip and strides == 1 and filters_in == filters_out:
+            if drop_rate > 0:
+                x = layers.Dropout(
+                    drop_rate, noise_shape=(None, 1, 1, 1), name=name + "drop"
+                )(x)
+            x = layers.add([x, inputs], name=name + "add")
+        return x
+
+    return apply
+
+
+def EfficientNet(
+    include_rescaling,
+    include_top,
+    width_coefficient,
+    depth_coefficient,
+    default_size,
+    dropout_rate=0.2,
+    drop_connect_rate=0.2,
+    depth_divisor=8,
+    activation="swish",
+    blocks_args="default",
+    model_name="efficientnet",
+    weights=None,
+    input_shape=(None, None, 3),
+    pooling=None,
+    num_classes=None,
+    classifier_activation="softmax",
+):
+    """Instantiates the EfficientNet architecture using given scaling coefficients.
+
+    Args:
+        include_rescaling: whether or not to Rescale the inputs.If set to True,
+                inputs will be passed through a `Rescaling(1/255.0)` layer.
+        include_top: whether to include the fully-connected
+            layer at the top of the network.
+        width_coefficient: float, scaling coefficient for network width.
+        depth_coefficient: float, scaling coefficient for network depth.
+        default_size: integer, default input image size.
+        dropout_rate: float, dropout rate before final classifier layer.
+        drop_connect_rate: float, dropout rate at skip connections.
+        depth_divisor: integer, a unit of network width.
+        activation: activation function.
+        blocks_args: list of dicts, parameters to construct block modules.
+        model_name: string, model name.
+        weights: one of `None` (random initialization),
+            or the path to the weights file to be loaded.
+        input_shape: optional shape tuple,
+            It should have exactly 3 inputs channels.
+        pooling: optional pooling mode for feature extraction
+            when `include_top` is `False`.
+            - `None` means that the output of the model will be
+                the 4D tensor output of the
+                last convolutional layer.
+            - `avg` means that global average pooling
+                will be applied to the output of the
+                last convolutional layer, and thus
+                the output of the model will be a 2D tensor.
+            - `max` means that global max pooling will
+                be applied.
+        num_classes: optional number of num_classes to classify images
```

Comment:
we renamed `num_classes` to `classes` recently for API consistency, lets make that change here!